### ISSUE TITLE
Remove liquibase.version from parent pom.xml

### ISSUE THREAD

**Issue by vramik** [keycloak/keycloak#22024]
### Description

With usage of [quarkus-bom](https://github.com/keycloak/keycloak/issues/20570) we could start leverage liquibase.version from there and stop overriding it in `pom.xml`.

**Comment by vramik** [comment]
Relates to (duplicate of?) https://github.com/keycloak/keycloak/issues/21392

**Comment by vmuzikar** [comment]
Yeah, I think this is a duplicate of https://github.com/keycloak/keycloak/issues/21392 :)

**Comment by vmuzikar** [comment]
jFTR, [test failures](https://github.com/keycloak/keycloak/actions/runs/5411587832/jobs/9834779175) after our initial attempt for removing the liquibase dependency from the root POM.

**Comment by sguilhen** [comment]
@vmuzikar just to make sure we're on the same page - the failures you linked were caused by the removal of httpclient, not liquibase, right? I was going through #21392 and it seemed to be the case.

**Comment by Pepo48** [comment]
@vmuzikar - correct. During the dependency cleanup task I tried to remove both httpclient and liquibase dependencies. IIRC, httpclient caused some unit test failures and liquibase some integration test failures.

One way or another, both removals were problematic.

@sguilhen the title of my issue is what might've confused you - the issue was renamed earlier today by Vašek (originally the title mention both the dependencies). I guess the reason is to address these two separately.

**Comment by sguilhen** [comment]
@Pepo48 Ah ok, thanks for clarifying!

**Comment by ahus1** [comment]
I don't see the liquibase.version in Keycloak's parent pom.xml, so I assume this has been resolved.
